samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/util/comparator/ImageMapGradingItemComparator.java

```diff
@@ -0,0 +1,24 @@
+package org.sakaiproject.tool.assessment.util.comparator;
+
+import java.util.Comparator;
+import org.sakaiproject.tool.assessment.data.dao.grading.ItemGradingData;
+
+/**
+ * This sorts ItemGrading items by itemGradingId.
+ */
+
+public class ImageMapGradingItemComparator implements Comparator<ItemGradingData> {

```

Comment:
This comparator should have a unit test and I would also look at using:
org.springframework.util.comparator.NullSafeComparator
as it's accounts for nulls already.

Reply:
Done!

Reply:
@ern All changes has been done. Please merge